### PR TITLE
[release-0.4] update cluster-api version to v0.3.19

### PR DESCRIPTION
**Pull request by nader-ziada**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
update cluster-api version to v0.3.19

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
update cluster-api version to v0.3.19
```

### CODE REVIEW

File: util/system/namespace_test.go [hunk 1]
```diff
@@ -44,9 +44,9 @@ func TestGetNamespace(t *testing.T) {
 	for _, c := range cases {
 		c := c
 		t.Run(c.Name, func(t *testing.T) {
-			t.Parallel()
 			g := gomega.NewWithT(t)
 			os.Setenv(NamespaceEnvVarName, c.PodNamespace)
+			defer os.Unsetenv(NamespaceEnvVarName)
```
Comment (nader-ziada) [inline util/system/namespace_test.go:49]: test was failing so needed to fix it

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

**Pull request merged**